dingo-net-netty/src/main/java/io/dingodb/net/netty/api/ApiProxy.java

```diff
@@ -117,6 +117,12 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
             EXEC.throwFormatError("invoke api on remote server", currentThread().getName(), e.getMessage());
         } catch (TimeoutException e) {
             EXEC_TIMEOUT.throwFormatError("invoke api on remote server", currentThread().getName(), e.getMessage());
+        } finally {
+            try {
+                channel.close();
```

Comment:
the channel is `open` outside the function, but `close` internal,  it looks strange.